docs/team/tanyutao544.md

```diff
@@ -11,20 +11,37 @@ SoC InternApply - SoC InternApply helps CS students to track internship applicat
 
 Given below are my contributions to the project.
 
-* **New Feature**:
+###New Feature Developed:
 
-* **New Feature**: 
+Below is a summary of features developed:
 
-* **Code contributed**: 
+####Details field
 
-* **Project management**:
+An additional field for users to add optional details for the internship application.
+Coded to use existing commands, reducing the application's complexity.
+Respective UI element developed to display the new field.
 
-* **Enhancements to existing features**:
+####Tag UI Component
 
-* **Documentation**:
+Developed the Ui component to the various Tags, like Priority, application status and Regular tags, differentiating each 
+tag type with a different color. The Ui component takes in the tagset stored in the application
+and sets the tag color respectively based on the tag.
 
-* **Community**:
+**Code contributed**: 
+[Link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=T11&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-02-18&tabOpen=true&tabType=authorship&tabAuthor=tanyutao544&tabRepo=AY2122S2-CS2103T-T11-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+###Project management:
 
-* **Tools**:
 
+###Documentation:
 
+####Developer Guide
+
+Introduction section of Developer guide, detailing suggested way to use the developer guide. 
+Method to navigate back to table of contents which is also used in the User Guide. Details field implementation explanation.
+Refactoring the UML diagrams in the logic section, to remove references to AddressBook3. Formatting the User stories into 
+a table.
+
+####User Guide
```

Comment:
As above.

Reply:
Odd, on my IDE, the preview displayed it correctly.